scheduler/bin/run-docker.sh

```diff
@@ -101,6 +101,21 @@ else
     COOK_ZOOKEEPER_LOCAL=true
 fi
 
+DATA_LOCAL_IP=$(docker inspect data-local | jq -r '.[].NetworkSettings.IPAddress')
+if [ "${DATA_LOCAL_IP}" = "null" ];
+then
+    echo "Starting data local server"
+    PROJECT_ROOT="$( dirname ${SCHEDULER_DIR} )"
+    ${PROJECT_ROOT}/integration/bin/run-data-local-server.sh &
+    sleep 5
```

Comment:
Should we use a `wait-for` strategy with a timeout and ping on port `5000` for a 200 response instead of a sleep? My concern is whether 5 seconds is not enough and if we later increase it whether we could have let go sooner.